<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>publications | Mahmoud  Bentriou</title>
    <meta name="author" content="Mahmoud  Bentriou">
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">
    <meta name="keywords" content="academic-website, portfolio-website, research, mahmoud, bentriou, bentriom, mahmoud-bentriou, bentrioum">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Bootstrap Table -->
    <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://bentriom.github.io/publications/">

    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Mahmoud </span>Bentriou</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/cv/">cv</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/repositories/">repositories</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">

<!--
-->

<ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Chounta2023" class="col-sm-8">
        <!-- Title -->
        <div class="title">The risk of valvular heart disease in the french childhood cancer survivors’ study: contribution of dose-volume histogram parameters</div>
        <!-- Author -->
        <div class="author">
        

        Stefania Chounta, Sarah Lemler, Nadia Haddy, and
          <span class="more-authors" title="click to view 16 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '16 more authors' ? 'Brice Fresneau, Imene Mansouri, Mahmoud Bentriou, Charlotte Demoor-Goldschmidt, Ibrahima Diallo, Vincent Souchard Msc, Duyen DO Thi, Cristina Veres, Aurore Surun, François Doz, Damien LLanas, Giao Vu-Bezin, Carole Rubino, Florent Vathaire, Véronique Letort, Rodrigue S. Allodji' : '16 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">16 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Radiotherapy and Oncology</em>, Jan 2023
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/J.RADONC.2023.109479"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1016/J.RADONC.2023.109479" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Background and purpose: Valvular Heart Disease (VHD) is a known complication of childhood cancer after radiotherapy treatment. However, the dose-volume-effect relationships have not been fully explored.
Materials and methods: We obtained individual heart Dose Volume Histograms (DVH) for survivors of the French Childhood Cancer Survivors Study (FCCSS) who had received radiotherapy. We calculated the Mean Dose to the Heart (MHD) in Gy, as well as the heart DVH parameters (Vd Gy, which represents the percentage of heart volume receiving at least d Gy), fixing the thresholds to 0.1 Gy, 5 Gy, 20 Gy, and 40 Gy. We analyzed them furtherly in the subpopulation of the cohort that was treated with a dose lower than 5 Gy (V0.1Gy|V5Gy=0%), 20 Gy (V5Gy|V20Gy=0%), and 40 Gy (V20Gy|V40Gy=0%), respectively. We inves- tigated their role in the occurrence of a VHD in this population-based observational cohort study using the Cox proportional hazard model, adjusting for age at cancer diagnosis and chemotherapy exposure. Results: Median follow-up was 30.6 years. Eighty-one patients out of the 7462 (1 %) with complete data experienced a severe VHD (grade ! 3). The risk of VHD increased along with the MHD, and it was asso- ciated with high doses to the heart (V40Gy &lt; 50 %, hazard ratio (HR) = 7.96, 95 % CI: 4.26–14.88 and V20Gy|V40Gy=0% &gt;50 %, HR = 5.03, 95 % CI: [2.35–10.76]). Doses 5–20 Gy to more than 50 % (V5Gy|V20Gy=0% &gt;50 %) of the heart induced a marginally non-significant estimated risk. We also observed a remarkable risk increase with attained age.
Conclusions: Our results provide new insight into the VHD risk that may impact current treatments and long-term follow-up of childhood cancer survivors.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Bentriou2022ACML" class="col-sm-8">
        <!-- Title -->
        <div class="title">Image based dosimetric features for the risk assessment of cardiac disease from childhood cancer therapy</div>
        <!-- Author -->
        <div class="author">
        

        <em>Mahmoud Bentriou</em>, Stefania Chounta, Rodrigue Allodji, and
          <span class="more-authors" title="click to view 7 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '7 more authors' ? 'Sarah Lemler, Duyen Thi Do, Florent Vathaire, Ibrahima Diallo, Stergios Christodoulidis, Maria Vakalopoulou, Véronique Letort-Le Chevalier' : '7 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">7 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Dec 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span>
              <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Hermange2022" class="col-sm-8">
        <!-- Title -->
        <div class="title">Inferring the initiation and development of myeloproliferative neoplasms</div>
        <!-- Author -->
        <div class="author">
        

        Gurvan Hermange, Alicia Rakotonirainy, <em>Mahmoud Bentriou</em>, and
          <span class="more-authors" title="click to view 7 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '7 more authors' ? 'Amandine Tisserand, Mira El-Khoury, François Girodon, Christophe Marzac, William Vainchenker, Isabelle Plo, Paul Henry Cournede' : '7 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">7 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Proceedings of the National Academy of Sciences of the United States of America</em>, Sep 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/PNAS.2120374119/SUPPL_FILE/PNAS.2120374119.SAPP.PDF"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1073/PNAS.2120374119/SUPPL_FILE/PNAS.2120374119.SAPP.PDF" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The developmental history of blood cancer begins with mutation acquisition and the resulting malignant clone expansion. The two most prevalent driver mutations found in myeloproliferative neoplasms-JAK2V617F and CALRm-occur in hematopoietic stem cells, which are highly complex to observe in vivo. To circumvent this difficulty, we propose a method relying on mathematical modeling and statistical inference to determine disease initiation and dynamics. Our findings suggest that CALRm mutations tend to occur later in life than JAK2V617F. Our results confirm the higher proliferative advantage of the CALRm malignant clone compared to JAK2V617F. Furthermore, we illustrate how mathematical modeling and Bayesian inference can be used for setting up early screening strategies.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Bentriou2021Thesis" class="col-sm-8">
        <!-- Title -->
        <div class="title">Statistical Inference and Verification of Chemical Reaction Networks</div>
        <!-- Author -->
        <div class="author">
        

        <em>Mahmoud Bentriou</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em></em> Dec 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span>
              <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Chemical Reaction Networks (CRN) constitute a formalism used to model biological processes. When the population number is not significant and the system is well-stirred, a Continuous-Time Markov Chain describes its stochastic dynamics. This class of model is characterised by the memoryless property: the future state of the system only depends on the current state.Statistical inference of such CTMCs is complex: likelihood computations are generally intractable. Approximate Bayesian Computation is a recent class of likelihood-free methods for Bayesian inference that allows approximating the posterior distribution with Monte Carlo simulations. It has proven its efficiency in the case of CTMCs.Model-checking was initially developed for assessing hardware and software systems’ reliability. There is a growing interest in the verification of models from Systems Biology to understand the complex molecular interactions within a biological system. Unfortunately, the state space of a CTMC modelled by a CRN quickly explodes or is infinite, which renders its complete exploration infeasible in practice. Statistical model checking methods have been developed to overcome this issue. They simulate the model and compute the ratio of simulations that fulfils a property. Recently, Hybrid Automata Stochastic Logic (HASL) has been introduced for the statistical verification of stochastic models. This temporal logic inherently adopts the statistical point of view of model checking.In this thesis, we focus on statistical inference and verification of CTMCs defined by CRNs. Our main contribution consists in the new formulation of an Approximate Bayesian Computation procedure combined with HASL called automaton-ABC. We apply this high-level method on several tasks of statistical inference and verification for biological CTMCs, including oscillatory models and time-bounded reachability problems. The implementation of our algorithms is documented and has led to a package in the Julia Programming language.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Bentriou2021" class="col-sm-8">
        <!-- Title -->
        <div class="title">Automaton-ABC: A statistical method to estimate the probability of spatio-temporal properties for parametric Markov population models</div>
        <!-- Author -->
        <div class="author">
        

        <em>Mahmoud Bentriou</em>, Paolo Ballarini, and Paul Henry Cournède</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Theoretical Computer Science</em>, Nov 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/J.TCS.2021.09.039"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1016/J.TCS.2021.09.039" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We present an adaptation of the Approximate Bayesian Computation method to estimate the satisfaction probability function of a temporal logic property for Markov Population Models. In this paper, we tackle the problem of estimating the satisfaction probability function of a temporal logic property w.r.t. a parametric Markovian model of Chemical Reaction Network. We want to assess the probability with which the trajectories generated by a parametric Markov Population Model (MPM) satisfy a logical formula over the whole parameter space. In the first step of the work, we formally define a distance between a trajectory of an MPM and a logical property. If the distance is 0, the trajectory satisfies the property. The larger the distance is, further the trajectory is from satisfying the property. In the second step, we adapt the Approximate Bayesian Computation method using the distance defined in the first step. This adaptation yields a new algorithm, called automaton-ABC, whose output is a density function that directly leads to the estimation of the desired satisfaction probability function. We apply our methodology to several examples and models, and we compare it to state-of-the-art techniques. We show that the sequential version of our algorithm relying on ABC-SMC leads to an efficient exploration of the parameter space with respect to the formula and gives good approximations of the satisfaction probability function at a reduced computational cost.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="LeRoux2020" class="col-sm-8">
        <!-- Title -->
        <div class="title">Comprehensive landscape of immune-checkpoints uncovered in clear cell renal cell carcinoma reveals new and emerging therapeutic targets</div>
        <!-- Author -->
        <div class="author">
        

        Diana Tronik-Le Roux, Mathilde Sautreuil, <em>Mahmoud Bentriou</em>, and
          <span class="more-authors" title="click to view 9 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '9 more authors' ? 'Jérôme Vérine, Maria Belén Palma, Marina Daouya, Fatiha Bouhidel, Sarah Lemler, Joel LeMaoult, François Desgrandchamps, Paul Henry Cournède, Edgardo D. Carosella' : '9 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">9 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cancer Immunology, Immunotherapy</em>, Jul 2020
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/S00262-020-02530-X/FIGURES/6"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/S00262-020-02530-X/FIGURES/6" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Clear cell renal cell carcinoma (ccRCC) constitutes the most common renal cell carcinoma subtype and has long been recognized as an immunogenic cancer. As such, significant attention has been directed toward optimizing immune-checkpoints (IC)-based therapies. Despite proven benefits, a substantial number of patients remain unresponsive to treatment, suggesting that yet unreported, immunosuppressive mechanisms coexist within tumors and their microenvironment. Here, we comprehensively analyzed and ranked forty-four immune-checkpoints expressed in ccRCC on the basis of in‐depth analysis of RNAseq data collected from the TCGA database and advanced statistical methods designed to obtain the group of checkpoints that best discriminates tumor from healthy tissues. Immunohistochemistry and flow cytometry confirmed and enlarged the bioinformatics results. In particular, by using the recursive feature elimination method, we show that HLA-G, B7H3, PDL-1 and ILT2 are the most relevant genes that characterize ccRCC. Notably, ILT2 expression was detected for the first time on tumor cells. The levels of other ligand-receptor pairs such as CD70:CD27; 4-1BB:4-1BBL; CD40:CD40L; CD86:CTLA4; MHC-II:Lag3; CD200:CD200R; CD244:CD48 were also found highly expressed in tumors compared to adjacent non-tumor tissues. Collectively, our approach provides a comprehensible classification of forty-four IC expressed in ccRCC, some of which were never reported before to be co-expressed in ccRCC. In addition, the algorithms used allowed identifying the most relevant group that best discriminates tumor from healthy tissues. The data can potentially assist on the choice of valuable immune-therapy targets which hold potential for the development of more effective anti-tumor treatments.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Bentriou2019" class="col-sm-8">
        <!-- Title -->
        <div class="title">Reachability Design Through Approximate Bayesian Computation</div>
        <!-- Author -->
        <div class="author">
        

        <em>M. Bentriou</em>, P. Ballarini, and P.-H. Cournède</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Jul 2019
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/978-3-030-31304-3_11"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1007/978-3-030-31304-3_11" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Time-bounded reachability problems are concerned with assessing whether a model’s trajectories traverse a given region of the state-space within given time-bounds. In the case of stochastic models reachability is associated with a measure of probability which depends on the model’s parameters. In this paper we propose a methodology that, given a reachability specification (for a parametric stochastic model), allows for computing a reachability related probability distribution on the parameter space, i.e. a distribution that allows for identifying regions of the parameter space for which there is a non-null probability to match the considered reachability specification. The methodology relies on the characterisation of distance between a model’s trajectory and a reachability specification which we show being assessable by using a hybrid automaton as a monitor of a model’s trajectory. An automata-based adaptation of the Approximated Bayesian Computation method is then introduced to estimate the reachability distribution on the parameter space.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"></div>

        <!-- Entry bib key -->
        <div id="Bentriou2018" class="col-sm-8">
        <!-- Title -->
        <div class="title">Assimilation de données par filtrage particulaire régularisé dans un modèle d’épidémiologie.</div>
        <!-- Author -->
        <div class="author">
        

        <em>Mahmoud Bentriou</em>, Stefanella Boatto, Gautier Viaud, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Catherine Bonnet, Paul-Henry Cournède' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Jul 2018
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
          </div>
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span>
              <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          
        </div>
      </div>
</li>
</ol>

</div>

          </article>

        </div>

      
    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2023 Mahmoud  Bentriou. <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> theme: <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Bootstrap Table -->
  <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script>

  <!-- Load Common JS -->
  <script src="/assets/js/no_defer.js"></script>
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
